src/wp-includes/widgets.php

```diff
@@ -1457,22 +1457,22 @@ function wp_map_sidebars_widgets( $existing_sidebars_widgets ) {
 }
 
 /**
- * Compares a list of sidebars with their widgets against a whitelist.
+ * Compares a list of sidebars with their widgets against an allowed list.
  *
  * @since 4.9.0
  *
  * @param array $sidebars_widgets List of sidebars and their widget instance IDs.
- * @param array $whitelist        Optional. List of widget IDs to compare against. Default: Registered widgets.
- * @return array Sidebars with whitelisted widgets.
+ * @param array $allowedlist      Optional. List of widget IDs to compare against. Default: Registered widgets.
+ * @return array Sidebars with allowed widgets.
  */
-function _wp_remove_unregistered_widgets( $sidebars_widgets, $whitelist = array() ) {
-	if ( empty( $whitelist ) ) {
-		$whitelist = array_keys( $GLOBALS['wp_registered_widgets'] );
+function _wp_remove_unregistered_widgets( $sidebars_widgets, $allowedlist = array() ) {
```

Comment:
Maybe `$allowed_widgets`? or `$allowed_widget_ids`?

Reply:
I like that! 👍🏼